nimare/meta/kernel.py

```diff
@@ -206,9 +206,9 @@ def transform(self, dataset, masker=None, return_type="image"):
         # Loop over exp ids since sparse._coo.core.COO is not iterable
         for i_exp, id_ in enumerate(transformed_maps[1]):
             if isinstance(transformed_maps[0][i_exp], sparse._coo.core.COO):
+                # This step is slow, but it is here just in case user want a
+                # return_type = "array", "image", or "dataset"
```

Comment:
Hm... should it be `if isinstance(...) and return_type != "sparse"`?

Reply:
I think if `return_type == "sparse"` this line won't be reached, the transformer will return the transformed_maps in line 202:

https://github.com/JulioAPeraza/NiMARE/blob/bb9815891727cac66cbda5a80da60d7e0ad42c9b/nimare/meta/kernel.py#L202-L203

Reply:
Ah, that makes sense.